charts/gardener/controlplane/values.yaml

```diff
@@ -284,6 +284,12 @@ global:
         # issuer: https://issuer.gardener.cloud
         minExpiration: 1h
         maxExpiration: 48h
+        signing:
+        # secretName: workload-identity-secret # must contain 'key.pem' key
+          key: |
+            -----BEGIN RSA PRIVATE KEY-----
+            ...
+            -----END RSA PRIVATE KEY-----
```

Comment:
I guess this block should be commented since it's optional to provide the key? Otherwise, people would submit this as key which makes the GAPI deployment probably go into a crashloop?

Reply:
Done.